python/tests/test_pipeline_builder.py

```diff
@@ -37,8 +37,8 @@ def test_local(self):
             TEST_CLIENT, name="notebook", sql=sql
         ).create_or_replace()
 
-        df_students = pd.read_csv("students.csv")
-        df_grades = pd.read_csv("grades.csv")
+        df_students = pd.read_csv("tests/students.csv")
```

Comment:
We have to also update the README to not `cd` into the tests directory.
https://github.com/feldera/feldera/blob/69d5f74cfe7997d80d8103d57b92209dbd730db9/python/README.md?plain=1#L67

Reply:
> Lets use the changes to this commit and update the readme for `pytest` to be: ` (cd python && python3 -m pytest tests)`.

I pushed an additional commit for that, does it look correct?